vxquery-rest/src/test/resources/logging.properties

```diff
@@ -0,0 +1,76 @@
+#
```

Comment:
This file doesn't need to be committed. I used it locally for handling logs. I will remove it from git.